tests/bson/bson-encode-004.phpt

```diff
@@ -22,7 +22,7 @@ $hannes->addFriend($mikola);
 
 var_dump($hannes);
 
-$s = fromArray(array($hannes));
+$s = fromArray($hannes);
```

Comment:
Not a fan. it causes bigass changes in the diff...
Is there a compelling reason to do this?
If you want to add the second variant as a new test case, thats cool - but I'm not a fan of manipulating the existing testcase in this way.